app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -101,6 +110,7 @@ def alternate_machine_id(machine_id)
         # providers store it in downcase, so, we let the upcase/downcase logic to other methods with more
         # business knowledge.
         # 1 - https://bugzilla.redhat.com/show_bug.cgi?id=1294461
+        return if machine_id.length < 31
```

Comment:
What changed in this PR that is causing a different machine_id to be passed to this method that is making this extra check necessary?